eventually-util/src/optional.rs

```diff
@@ -66,7 +66,7 @@ pub trait Aggregate {
     /// a _newtype instance_ compatible with the core
     /// [`Aggregate`](eventually_core::aggregate::Aggregate) trait.
     #[inline]
-    fn as_aggregate(self) -> AsAggregate<Self>
+    fn into_aggregate(self) -> AsAggregate<Self>
```

Comment:
I like this, and I don't know why I haven't followed [the Naming guideline](https://rust-lang.github.io/api-guidelines/naming.html?highlight=into_#ad-hoc-conversions-follow-as_-to_-into_-conventions-c-conv) before 🤦🏻‍♂️ 

However, if you're changing the method name here, would you mind also renaming the type from `AsAggregate` to `IntoAggregate`? Just to keep it consistent 😉

Reply:
:+1: